### DISCUSSION TITLE
Add support to download a copy of a shared note into my vault

### DISCUSSION THREAD

**Discussion by jvsteiner** [alangrainger/share-note#133] [Vote for upcoming features]
I think there is a use case, for when there are two users and both have Share Note installed.  As the recipient of a shared note, I would like to be able to download a copt of the shared note into my vault, and then maybe make some modification to it.  Perhaps I will reshare it with the original author or maybe someone else.  I realize there is already a feature request for editing and comments, but this is really different.  It's less about providing edits and feedback, although, if I reshare to the original author, that is indeed what might be happening.  Instead, this feature is more about the ability to pass a note to someone - who can then take it over, or use it for their own purposes. 

I'm explicitly not suggesting that the note needs to be synchronized between all shared users.  Although that sounds cool, I think it's much too complicated, and unnecessary for the kind of situation many people might have.

**Comment by jvsteiner** [comment]
I made a proof of concept plugin from a fork that can be used with BRAT here: https://github.com/jvsteiner/send-note

**Reply by huyz** [reply]
@jvsteiner your POC looks very useful. Is it meant to be used or is it really just a POC?

**Comment by huyz** [comment]
I was going to make a feature request which is along the same lines—perhaps, one could consider it a stepping stone to what you're asking @jvsteiner.

I wanted the Markdown source to be published alongside the rendered note.  Perhaps, all the recipient has to do is manually add `.md` to the URL (before any `#encryption-key` part). Of course, this could be easily handled by a bookmarklet installed by the recipient if editing URLs becomes a regular thing and too burdensome.

What do you think about this @alangrainger as a simple stepping stone / workaround for OP's feature request?  Should I file a separate FR?

**Reply by alangrainger** [reply]
I like this idea a lot! Should be easy enough to implement, I'll give it a go this week.

**Reply by graidl** [reply]
I second this! It should be a quite simple extension that would be worth much!

**Comment by jvsteiner** [comment]
I went ahead and made it as a separate plugin because i needed it. I use it personally - helping my son with his maths homework remotely- so its not exactly a serious commercial venture, but it works and is available in the plugin directory. Feel free to steal code if you want to - you are very welcome to it, and if, in the end , all the functionality is available in one plugin then so much the better. I currently have about 50 installed and am maintaining 4, so I am happy to reduce both numbers…

**Reply by jvsteiner** [reply]
Plugin is available as send-note

**Comment by chrsdk** [comment]
Any updates on this? This is actually what I am looking for in this plugin, sending my notes to family members just as they are in my vault so that they can effortlessly add these notes in their vaults.

**Reply by jvsteiner** [reply]
https://github.com/jvsteiner/send-note is still available in the community plugins list, despite recent changes to obsidian plugin requirements, its still there

**Comment by alangrainger** [comment]
This has been added in 1.6.0. Thanks @jvsteiner for the suggestion, and @chrsdk @graidl for following up on it. Apologies for taking so long :\

Docs here: https://docs.note.sx/notes/importing-shared-notes
